src/components/DiffView/styles.module.scss

```
@@ -60,3 +60,7 @@
     background-color: $light-gray;
   }
 }
+
+.inlineLinterMessages {
+  padding: 16px 16px 0 16px;
```

Bootstrap sets `margin-botom: 1rem` to the alert component, so I guess we should also use `1rem` here:

```suggestion
  padding: 1rem 1rem 0 1rem;
```

---

Thanks